concourse/scripts/common.sh

```
@@ -2240,11 +1738,11 @@ function run_eloqkv_cluster_tests() {
       redis_pids+=($!)
       index=$((index + 1))
     done
-    echo "redis_servers with nowal, nostore, 36000 ckpt interval are started, pids: $redis_pids"  >> /tmp/redis_cluster_with_eloqstore.log
+    echo "redis_servers with nowal, nostore, 36000 ckpt interval are started, pids: $redis_pids" >>/tmp/redis_cluster_with_eloqstore.log
```

Shellcheck (0.11.0)

[warning] Expanding an array without an index only gives the first element.

(SC2128)